--- kibana-4.0.1-linux-x64/src/public/index.js
+++ kibana-4.0.1-linux-x64_huh/src/public/index.js
@@ -118965,9 +118965,9 @@
       .sort(getSort($state.sort, $scope.indexPattern))
       .query(!$state.query ? null : $state.query)
       .highlight({
-        pre_tags: [highlightTags.pre],
-        post_tags: [highlightTags.post],
-        fields: {'*': {}}
+        //pre_tags: [highlightTags.pre],
+        //post_tags: [highlightTags.post],
+        //fields: {'*': {}}
       })
       .set('filter', $state.filters || []);
     });
